nekoyume/Assets/_Scripts/UI/Tutorial/TutorialController.cs

```diff
@@ -158,7 +158,7 @@ private static T GetData<T>(string path) where T : new()
             return data;
         }
 
-        private static int GetCheckPoint(int clearedStageId)
+        public static int GetCheckPoint(int clearedStageId)
```

Comment:
II think it can be replaced with TutorialController.IsPlaying.

Reply:
If it doesn't check checkPoint and TutorialController.IsPlaying both, Notice Popup will be shown buring tutorial!